internal/logger/target/http/http.go

```diff
@@ -343,27 +340,25 @@ func (h *Target) startQueueProcessor(ctx context.Context, mainWorker bool) {
 	globalBuffer := logChBuffers[name]
 	logChLock.Unlock()
 
-	newTicker := time.NewTicker(time.Second)
-	isTick := false
 	var count int
-
 	for {
-		isTick = false
-		select {
-		case _ = <-newTicker.C:
-			isTick = true
-		case entry, _ = <-globalBuffer:
-		case entry, ok = <-h.logCh:
-			if !ok {
+		var (
+			ok    bool
+			entry any
+		)
+
+		if count < h.batchSize {
+			select {
```

Comment:
It's probably not a good idea to remove the ticker. If someone configures a batch size of lets say 100 but their system is having a slow day, then those audit logs will sit in the queue until they reach 100.  It creates a bigger time window where audit logs could be lost due to hard restarts and we'll probably receive tickets about log delivery being slow from some clients.